packs/hpe-icsp/actions/icsp_job_status_get.yaml

```diff
@@ -0,0 +1,26 @@
+---
+  name: "icsp_job_status_get"
+  runner_type: "run-python"
+  description: "Return Status of specified Job"
+  enabled: true
+  entry_point: "icsp_job_status_get.py"
+  parameters:
+    job_id:
+      type: "integer"
+      description: "URI for Job, blank will return all jobs listed"
+      required: false
+      position: 0
+    monitor:
+      type: boolean
+      description: "Monitor Progress of The Job. False = One time check"
+      required: false
+      default: false
```

Comment:
When you mentiond default arguments which parameter do you mean?
Job id can be empty. and the default for monitor is supposed to be "false"
Unless it's a requirement in my head monitor sounds more fitting to poll. but if it's that important i can adjust.